README.md

```diff
@@ -146,13 +190,16 @@ posts for further information
     </td><td><code>Standard_A2</code></td></tr>
 
   <tr><td>kibanaCertBlob</td><td>string</td>
-    <td>A Base-64 encoded form of the certificate (.crt) to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
+    <td>A Base-64 encoded form of the certificate (.crt) in PEM format to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
 
   <tr><td>kibanaKeyBlob</td><td>securestring</td>
-    <td>A Base-64 encoded form of the private key (.key) to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
+    <td>A Base-64 encoded form of the private key (.key) in PEM format to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
```

Comment:
Again PEM keys will always be Base64 encoded.

Reply:
Similar to https://github.com/elastic/azure-marketplace/pull/187#discussion_r201193041

I'll add an example to the README of how to pass certificates to the template, to make it clear.